mysqlsmo/__init__.py

```diff
@@ -3,7 +3,7 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
It looks good now, lets just change the name of the pg and MySQL server files to keep the naming consistent with the class name.